src/pages/community.tsx

```diff
@@ -0,0 +1,479 @@
+import { GetStaticProps } from "next"
+import { SSRConfig, useTranslation } from "next-i18next"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import {
+  Box,
+  Flex,
+  HeadingProps,
+  SimpleGrid,
+  Text,
+  useTheme,
+} from "@chakra-ui/react"
+
+import { ChildOnlyProp, CommonHeroProps } from "@/lib/types"
+import { ICard, IGetInvolvedCard } from "@/lib/interfaces"
+
+import { getLastDeployDate } from "@/lib/utils/getLastDeployDate"
+import { getRequiredNamespacesForPage } from "@/lib/utils/translations"
+
+import ActionCard from "../components/ActionCard"
+import ButtonLink from "../components/Buttons/ButtonLink"
+import Callout from "../components/Callout"
+import Card from "../components/Card"
+import FeedbackCard from "../components/FeedbackCard"
+import { HubHero } from "../components/Hero"
+import { Image } from "../components/Image"
+import OldHeading from "../components/OldHeading"
+import PageMetadata from "../components/PageMetadata"
+
+// Static assets
+import developersEthBlockImg from "@/public/developers-eth-blocks.png"
+import dogeComputerImg from "@/public/doge-computer.png"
+import ethImg from "@/public/eth.png"
+import financeTransparentImg from "@/public/finance_transparent.png"
+import futureTransparentImg from "@/public/future_transparent.png"
+import hackathonTransparentImg from "@/public/hackathon_transparent.png"
+// -- Hero
+import communityHeroImg from "@/public/heroes/community-hero.png"
+// -- Cards
+import upgradesCoreImg from "@/public/upgrades/core.png"
+import whatIsEthereumImg from "@/public/what-is-ethereum.png"
+
+type Props = SSRConfig & {
+  lastDeployDate: string
+}
+
+export const getStaticProps = (async (context) => {
+  const { locale } = context
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPage("/community")
+  const lastDeployDate = getLastDeployDate()
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      lastDeployDate,
+    },
+  }
+}) satisfies GetStaticProps<Props>
+
+const CardContainer = ({ children }: ChildOnlyProp) => {
+  return (
+    <Flex wrap="wrap" mx={-4}>
+      {children}
+    </Flex>
+  )
+}
+
+const Content = ({ children }: ChildOnlyProp) => {
+  return (
+    <Box py={4} px={8} w="full">
+      {children}
+    </Box>
+  )
+}
+
+const Divider = () => {
+  return <Box my={16} w="10%" h={1} bgColor="homeDivider" />
+}
+
+const Page = ({ children }: ChildOnlyProp) => {
+  return (
+    <Flex direction="column" alignItems="center" w="full" mx="auto">
+      {children}
+    </Flex>
+  )
+}
+
+const ButtonRow = ({ children }: ChildOnlyProp) => {
+  return (
+    <Flex alignItems="flex-start" direction={{ base: "column", md: "row" }}>
+      {children}
+    </Flex>
+  )
+}
+
+const StyledButtonLink = ({ children, ...props }: any) => {
```

Comment:
`ButtonLinkProps` from `@/components/Buttons` should work here

Reply:
Thanks, fixed